app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,23 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(
+            self,
+            nickname: str,
+            favourite_dish: str,
+            hummer_level: int
+    ) -> None:
+        super().__init__(nickname, favourite_dish)
+        self._hummer_level = hummer_level
+
+    def get_rating(self) -> int:
+        return self._hummer_level + 4
+
+    def player_info(self) -> str:
+        return (f"Dwarf warrior "
+                f"{self.nickname}. {self.nickname} has a hummer of the "
+                f"{self._hummer_level} level")
+
+    def eat_favourite_dish(self) -> None:
+        print(f"{self.nickname} is eating {self._favourite_dish}")
```

Comment:
This method is duplicated. You can remove it and implement it once in Dwarf class